src/utilities/network/lookupApi.js

```diff
@@ -22,6 +22,34 @@ const SEPARATORS = {
   POLICY_NUMBER: '-',
 };
 
+export const createInsurancePolicyRecord = response => {
```

Comment:
same comment as before - adding a case to `createRecord` should be what we're doing here!

Reply:
#2664 :).